day_7/checker_board.rb

```diff
@@ -0,0 +1,27 @@
+print "Board size: "
+size = gets.chomp.to_i
+range = (1..size)
+
+range.each do |n|
+  s = size
+  if n.even?
+    while s > 0
+      if s.even?
+        print "X"
+      else
+        print " "
+      end
+      s = s - 1
+    end
```

Comment:
Pro-tip: loops inside of loops* can often be abstracted into it's own method that you can then call.

*though nested loops are inefficient computationally, they're often necessary for 2D problems like this one.